javaagent-extension-api/src/main/java/io/opentelemetry/javaagent/extension/instrumentation/InstrumentationModule.java

```diff
@@ -100,6 +100,17 @@ public boolean isHelperClass(String className) {
     return false;
   }
 
+  /**
+   * Instrumentation modules that override this to true (recommended) will use the inline=false
```

Comment:
I suggest changing `will` to MUST or SHOULD to be a little more clear on the optionality.

Reply:
```suggestion
   * Instrumentation modules that override this to true (recommended) must use the inline=false
```